src/renderers/dom/client/utils/setInnerHTML.js

```diff
@@ -28,7 +31,20 @@ var createMicrosoftUnsafeLocalFunction = require('createMicrosoftUnsafeLocalFunc
  */
 var setInnerHTML = createMicrosoftUnsafeLocalFunction(
   function(node, html) {
-    node.innerHTML = html;
+    if ('innerHTML' in node) {
```

Comment:
Either would be fine. Namespace works better in jsdom but if you prefer instanceof SVGElement that seems fine too.